apps/backoffice-v2/src/common/components/organisms/Header/hooks/useNavbarLogic/useNavbarLogic.tsx

```diff
@@ -57,6 +63,10 @@ export const useNavbarLogic = () => {
   const { pathname } = useLocation();
   const checkIsActiveFilterGroup = useCallback(
     (navItem: TRouteWithChildren) => {
+      if (!navItem.children) {
+        return navItem.href === pathname;
+      } 
```

Comment:
Revert please. This should never be true since a filter group always has children. The suggestion in the previous comment was general advice for the future.